.github/workflows/patch-release-pr.yml

```diff
@@ -14,8 +14,8 @@ jobs:
   check:
     uses: "grafana/loki-release/.github/workflows/check.yml@loki-2.9.x"
     with:
-      build_image: "grafana/loki-build-image:0.30.1"
-      golang_ci_lint_version: "v1.51.2"
+      build_image: "grafana/loki-build-image:0.33.1"
+      golang_ci_lint_version: "v1.55.1"
```

Comment:
I guess this downgrade is to be able to work with the linter?

Reply:
It's an upgrade. It matches the linter used in the build image now.